freyja/1.3.12/Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM mambaorg/micromamba:0.24.0 as app
+
+# Version arguments
+# ARG variables only persist during build time
+ARG FREYJA_SOFTWARE_VERSION="1.3.12"
+
+# build and run as root users since micromamba image has 'mambauser' set as the $USER
+USER root
+# set workdir to default for building; set to /data at the end
+WORKDIR /
+
+LABEL base.image="mambaorg/micromamba:0.24.0"
+LABEL dockerfile.version="1"
+LABEL software="Freyja"
+LABEL software.version=${FREYJA_SOFTWARE_VERSION}
+LABEL description="Freyja is a tool to recover relative lineage abundances from mixed SARS-CoV-2 samples from a sequencing dataset (BAM aligned to the Hu-1 reference)"
+LABEL website="https://github.com/andersen-lab/Freyja"
+LABEL license="https://github.com/andersen-lab/Freyja/blob/main/LICENSE"
+LABEL maintainer1="Kevin Libuit"
+LABEL maintainer1.email="kevin.libuit@theiagen.com"
```

Comment:
This looks great! 

This dockerfile is going to fail testing, though, because we need to adjust maintainer1 to maintainer and maintainer1.email to maintainer.email (the rest can probably stay the same).